src/styles/styles.js

```diff
@@ -490,11 +550,42 @@ const styles = {
         textAlignVertical: 'center',
     },
 
+    expensiPickerContainer: {
+        borderWidth: 1,
+        borderRadius: variables.componentBorderRadiusNormal,
+        borderColor: themeColors.border,
+        justifyContent: 'center',
+        backgroundColor: themeColors.componentBG,
+    },
+    expensiPickerLabel: {
+        position: 'absolute',
+        left: 12,
+        top: 8,
+    },
+    expensiPicker: (disabled = false) => ({
+        iconContainer: {
+            top: 16,
+            right: 12,
+        },
+        inputWeb: {
+            appearance: 'none',
+            cursor: disabled ? 'not-allowed' : 'pointer',
+            border: 'none',
+            ...expensiPicker,
+        },
+        inputIOS: {
```

Comment:
Since these are the same style, do we need both? Can we just make it `inputNative`?

Reply:
I don't think `react-native-picker-select` supports `inputNative`
https://github.com/lawnstarter/react-native-picker-select/blob/ca6488c2eef5c776a8071400c8b7987712d43397/index.d.ts#L36-L40

Reply:
Okay, but in our code we can provide the same `inputNative` style object [here](https://github.com/Expensify/App/blob/6abfbb93f32e2b4a5e9493ed2beed6e762ac7e17/src/components/Picker/pickerDisabledStyles/index.android.js#L5) and [here](https://github.com/Expensify/App/blob/6abfbb93f32e2b4a5e9493ed2beed6e762ac7e17/src/components/Picker/pickerDisabledStyles/index.ios.js#L5)

Reply:
As soon as we get this thread resolved we are good to merge 👍

Reply:
I actually don't use `pickerDisabledStyles`.

and just to DRY this:
```js
inputIOS: {
    ...expensiPicker,
},
inputAndroid: {
    ...expensiPicker,
},
```

I'll have to create 4 files. `Picker/pickerStyles/index.js`, `Picker/pickerStyles/index.ios`, `Picker/pickerStyles/index.android.js`, `Picker/pickerStyles/propTypes.js`

_Picker/pickerStyles/index.android.js_ for example:

```js
import {propTypes, defaultProps} from './propTypes';
import styles from '../../../styles/styles';

const pickerStyles = disabled => ({
    ...styles.expensiPicker(disabled),
    inputAndroid: styles.expensiPicker(disabled).inputNative,
});

pickerStyles.propTypes = propTypes;
pickerStyles.defaultProps = defaultProps;
pickerStyles.displayName = 'pickerStyles';

export default pickerStyles;
```

Reply:
but ok.. I'll just do it

Reply:
I'm curious [is this](https://github.com/Expensify/App/pull/3414/commits/adeddc786b5a91dc7014ebf93c21390670bc4a24) worth it?